cpp/include/kvikio/file_handle.hpp

```diff
@@ -314,16 +314,26 @@ class FileHandle {
    * @param file_offset Offset in the file to read from.
    * @param devPtr_offset Offset relative to the `devPtr_base` pointer to read into.
    * This parameter should be used only with registered buffers.
+   * @param sync_null_stream Synchronize the CUDA null stream prior to calling cuFile. Contrary to
```

Comment:
Do we mean "default stream" instead of "null stream" everywhere here? https://docs.nvidia.com/cuda/cuda-runtime-api/stream-sync-behavior.html

Reply:
Right, I have renamed to default stream. cuFile calls it the [null stream](https://docs.nvidia.com/gpudirect-storage/api-reference-guide/index.html#cufile-io-api-functional-specification), but I agree default stream is less confusing.